### PR TITLE
Fix issues in TorpedoStore class

### PR DESCRIPTION
**Pull request by csinga24**
I fixed the 3 issue in TopreoStore class, what the SonerLint found.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (feliciakranitz) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:14]: Nice solution!

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
```
Comment (feliciakranitz) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Please remove unnecessary empty line

**Review by feliciakranitz — CHANGES_REQUESTED** [review]
You did a great job with this extremely complex and tricky task, but please, next time, have care with these ugly empty lines, they can be very dangerous!

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (csinga24) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:14]: Thank you

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (csinga24) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Thank, I fixed it already.

**Review by feliciakranitz — APPROVED** [review]

**Pull request merged**